Modules/Microsoft365DSC/DSCResources/MSFT_AADApplication/MSFT_AADApplication.psm1

```diff
@@ -163,6 +183,110 @@ function Get-TargetResource
         else
         {
             Write-Verbose -Message 'An instance of Azure AD App was retrieved.'
+
+
+            $AADBetaApp= Get-MgBetaApplication -Property "id,displayName,appId,authenticationBehaviors" -ApplicationId $ObjectID -ErrorAction SilentlyContinue
+            $AADAppKeyCredentials = Get-MgApplication -Property "keyCredentials" -ApplicationId $ObjectID -ErrorAction SilentlyContinue
```

Comment:
Isn't this already retrieved by line 171 above? Do we need to make yet another call across the wire? Can't we just do one single call to the beta call for the entire Get-TargetResource function?

Reply:
I am not sure if there are any changes in the format or supportability of other properties in beta cmdlet. So I chose to use beta to get only authenticationBehaviors property